app/presenters/api/v2/headings/declarable_heading_presenter.rb

```diff
@@ -59,6 +59,12 @@ def entry_price_system?
           import_measures.any?(&:entry_price_system?)
         end
 
+        def basic_duty_rate
+          if third_country_measures.one?
+            third_country_measures.first&.formatted_duty_expression
+          end
+        end
```

Comment:
It's not a big issue, but this method is duplicated.

Reply:
Yeah, I'm not even convinced we use these anywhere

Reply:
And by use anywhere I mean the frontend declares, but does not implement them. We've essentially replaced this with the import trade summary you built in 2021